src/app/user-page/user-page.component.ts

```diff
@@ -76,5 +79,8 @@ export class UserPageComponent extends Base implements OnInit {
 
   ngOnInit(): void {
     this.getUserInfo(this.username);
+    this.tokenQuery.tokens$.subscribe((tokens: TokenUser[]) => {
```

Comment:
I believe you need a `takeUntil`, or this will cause a leak. Or, don't subscribe at all, and use `async` in the HTML.